dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -120,13 +120,13 @@ ANNO_CLASSSOURCE_OPEN3_EXCEPTION.useraction=See the exception and the associated
 # # {4} The error message from the exception
 ANNO_CLASSSOURCE_CLOSE4_EXCEPTION=CWWKC0011W: [ {0} ]: The close of resource [{1}] under root [{2}] for class [{3}] failed. Error message: {4}
 ANNO_CLASSSOURCE_CLOSE4_EXCEPTION.explanation=The close of the InputStream for the resource caused an exception.
-ANNO_CLASSSOURCE_CLOSE4_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_CLOSE4_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
 
 # # {0} A hashcode id used to identify the object emitting the message
 # # {1} A jar file path name
 ANNO_CLASSSOURCE_OPEN4_EXCEPTION=CWWKC0012W: [ {0} ]: The open of [{1}] failed with an exception.
 ANNO_CLASSSOURCE_OPEN4_EXCEPTION.explanation=The file was being opened as a JarFile and that action caused an exception.
-ANNO_CLASSSOURCE_OPEN4_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_OPEN4_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated.